desktop/release-notes.md

```diff
@@ -24,12 +24,95 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.15.0
+
+{% include release-date.html date="2022-12-01" %}
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Substantial performance improvements for macOS users with the option of enabling the new VirtioFS file sharing technology. Available for macOS 12.5 and above. 
+- Docker Desktop for Mac no longer needs to install the privileged helper process `com.docker.vmnetd` on install or on the first run. For more information see [Permission requirements for Mac](https://docs.docker.com/desktop/mac/permission-requirements/).
+- Added [WebAssembly capabilities](wasm/index.md). Use with the [containerd integration](containerd/index.md).
+- Improved the descriptions for beta and experimental settings to clearly explain the differences and how people can access them.
+- Available disk space of VM now displays in the footer of Docker Dashboard.
+- A disk space warning now displays in the footer if available space is below 3GB.
+- Updated colours and typography to be in line with Docker Design System.
```

Comment:
Do we think people know what the Docker Design System is? Might it be better to instead state the purpose of the changes? Eg increased contrast to meet certain accessibility standards (a11y).

cc @rebecapereira-docker

Reply:
@joe0BAB maybe we can use what we did for the Updates Highlights instead:
`Changes to Docker Desktop's interface as we become more ADA accessible and visually unified.`